modules/migrations/gitea.go

```diff
@@ -263,30 +263,25 @@ func (g *GiteaLocalUploader) CreateReleases(releases ...*base.Release) error {
 			}
 
 			// download attachment
-			err = func() error {
-				resp, err := http.Get(asset.URL)
-				if err != nil {
-					return err
-				}
-				defer resp.Body.Close()
-
-				localPath := attach.LocalPath()
-				if err = os.MkdirAll(path.Dir(localPath), os.ModePerm); err != nil {
-					return fmt.Errorf("MkdirAll: %v", err)
-				}
-
-				fw, err := os.Create(localPath)
-				if err != nil {
-					return fmt.Errorf("Create: %v", err)
-				}
-				defer fw.Close()
-
-				_, err = io.Copy(fw, resp.Body)
+			resp, err := http.Get(asset.URL)
```

Comment:
The closure is to ensure `resp.Body.Close()` could be executed immediately.

Reply:
Fixed